tests/kernel_bundle/sycl_join_kernel_bundle_with_empty_one.cpp

```diff
@@ -30,13 +30,13 @@ template <sycl::bundle_state State>
 void run_verification(util::logger &log) {
   auto queue = util::get_cts_object::queue();
   const auto ctx = queue.get_context();
-  const auto dev = queue.get_device();
 
   auto kb = sycl::get_kernel_bundle<State>(ctx);
 
   // Selector that always returns false. Used to get empty kernel_bundle
   auto false_selector = [](const sycl::device_image<State> &) { return false; };
-  auto empty_kb = sycl::get_kernel_bundle<State>(ctx, {dev}, false_selector);
+  auto empty_kb =
+      sycl::get_kernel_bundle<State>(ctx, ctx.get_devices(), false_selector);
```

Comment:
```suggestion
  auto empty_kb = sycl::get_kernel_bundle<State>(ctx, false_selector);
```

List of devices can be omitted, it will be automatically taken from the context